app/__init__.py

```diff
@@ -34,9 +32,26 @@ def create_app(debug=False):
     # ----- Routes ----------
     app.register_blueprint(main_blueprint)
 
+    server_cfg = {}
     cfg_file = BASE_PATH.joinpath('config.yaml')
-    with open(cfg_file, 'r') as f:
-        server_cfg = yaml.safe_load(f)
+    if pathlib.Path(cfg_file).exists():

```

Comment:
I think its fine leaving the `if 'aliases' in server_cfg:` where it was... suggestion would be if file doesn't exist, copy `config.example.yaml` to `config.yaml` for the user so all they have to do is edit it.  Loading the template would just give the same behavior as leaving `server_cfg = {}`

Reply:
Yeah good idea, sure I'll make that as an update here tonight when I get a chance.

Reply:
I changed this so that if the expected config.yaml didn't exist upon server startup we will copy it directly into a config.yaml so we always have one.

Though I'd say it is fine if we just abort with an unhandled exception if both `config.example.yaml` and `config.yaml` don't exist as then the user running the server did something wrong.